.github/workflows/windows-maven.yaml

```diff
@@ -0,0 +1,30 @@
+name: Windows Maven build
+
+on: [pull_request,push]
+  paths-ignore:
+    - 'tailormap-components/**'
+
+jobs:
+  build:
+    name: Windows Java ${{ matrix.java }}
+    runs-on: [windows-latest]
+    strategy:
+      matrix:
+        java: [11]
+
+    steps:
+    - uses: actions/checkout@v2
+    - uses: actions/cache@v2
+      with:
+        path: ~/.m2/repository
+        key: ${{ runner.os }}-maven-${{ hashFiles('**/pom.xml') }}
+        restore-keys: |
+          ${{ runner.os }}-maven-
+    - name: Set up JDK
+      uses: actions/setup-java@v1
+      with:
+        java-version: ${{ matrix.java }}
+    - name: Build with Maven
+      run: |
+        mvn -B test -fae
+        mvn -B verify -fae -e
```

Comment:
ik denk niet dat dit doet wat je denkt, voor integratie tests is een profile nodig en extra opties

Reply:
Het was vooral een uitprobeersel, om te kijken of ik mssql hier werkend kon krijgen, zodat we appveyor niet meer nodig hadden, maar dat was vanmorgen een beetje gestrand, dus die kan weg

Reply:
Running Microsoft SQL Server 2019 Developer Edition in a Docker Windows Container

https://www.simteq.com/2020/05/14/running-microsoft-sql-server-2019-developer-edition-in-a-docker-windows-container/